### PR TITLE
Add integration test for security kibana plugin 

### PR DESCRIPTION
**Pull request by gaiksaya**
*Issue #, if available:*
P42915102, https://github.com/opendistro-for-elasticsearch/opendistro-build/issues/522

*Description of changes:*
- Add integration test for security kibana plugin.
- Make changes to installation scripts to add the setting as per conversation with security kibana team
- The test have been added with new folder structure in mind. No changes have been made to other tests
- Change version.py env to python instead of python3 as windows github actions is giving issues

*Test Results:*

- Windows: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/452139633
- RPM: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/451870374
- DEB: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/451798083
- TAR: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/451764566
- Docker: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/452159918

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/staging-build-deb.yml [hunk 1]
```diff
@@ -649,3 +649,69 @@ jobs:
         with:
           working-directory: kibana/plugins/anomaly-detection-kibana-plugin
           command: yarn cy:run-with-security --config baseurl=http://localhost:5601
+
+  Test-SEC-KIBANA:
+    needs: [sign-deb-artifacts]
+    runs-on: [ubuntu-16.04]
```
Comment (sreekarjami) [inline .github/workflows/staging-build-deb.yml:655]: Have you tried the workflow with ubuntu 18.x

File: .github/workflows/staging-build-rpm.yml [hunk 2]
```diff
@@ -587,9 +587,79 @@ jobs:
...
+      
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Get node and yarn versions
+        id: node_yarn_versions
+        run: |
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
```
Comment (sreekarjami) [inline .github/workflows/staging-build-rpm.yml:629]: Can we change nvm version to v0.37.2 , the v0.34.0 is release in 2019.

File: .github/workflows/staging-build-tar.yml [hunk 3]
```diff
@@ -555,3 +556,75 @@ jobs:
         with:
           working-directory: kibana/plugins/anomaly-detection-kibana-plugin
           command: yarn cy:run-with-security --config baseurl=http://localhost:5601
+
+  Test-SEC-KIBANA:
+    needs: [build-es-artifacts, build-kibana-artifacts]
+    runs-on: [ubuntu-16.04]
```
Comment (sreekarjami) [inline .github/workflows/staging-build-tar.yml:562]: Can we run on ubuntu 18.x

File: .github/workflows/staging-build-deb.yml (hunk 1 above)
Comment (gaiksaya) [reply .github/workflows/staging-build-deb.yml]: Changed to 18. It works https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/461658927

File: .github/workflows/staging-build-tar.yml (hunk 3 above)
Comment (gaiksaya) [reply .github/workflows/staging-build-tar.yml]: Changed to 18

File: .github/workflows/staging-build-rpm.yml (hunk 2 above)
Comment (gaiksaya) [reply .github/workflows/staging-build-rpm.yml:629]: Not required as of now. Its fine as it is just used to install node

**Review by peterzhuamazon — APPROVED** [review]

**Review by sreekarjami — APPROVED** [review]

**Pull request merged**